components/compliance-service/reporting/relaxting/suggestions.go

```diff
@@ -67,13 +69,15 @@ func (backend ES2Backend) GetSuggestions(typeParam string, filters map[string][]
 	useSummaryIndex := true
 	if len(filters["control"]) > 0 {
 		useSummaryIndex = false
-	} else {
-		// Going through all filters to find the ones prefixed with 'control_tag', e.g. 'control_tag:nist'
-		for filterType := range filters {
-			if strings.HasPrefix(filterType, "control_tag:") {
-				useSummaryIndex = false
-				break
-			}
+	}
+
+	controlTagFilterKey := ""
+	// Going through all filters to find the ones prefixed with 'control_tag', e.g. 'control_tag:nist'
+	for filterType := range filters {
+		if strings.HasPrefix(filterType, "control_tag:") {
+			_, controlTagFilterKey = leftSplit(filterType, ":")
+			useSummaryIndex = false
+			break
```

Comment:
Instead of embedding the value in the key name, why not use `control_tag` as the key and the string to the right of the colon as the value?

Reply:
Filters like environment, platform, node_id have one or multiple values. Contol tags are different because they have a key and one or multiple values.
We discussed this in one of the standups and decided to follow the pattern already used for the node manager and put the key in the filter type.

Reply:
I don't understand this comment. Why is it better to put a key and value into the key and then not provide a value?

In case I am not making any sense, why is this:

```
Reporting::ListFilter.new(type: 'control_tag:satisfies', values: []),
```

better than this?

```
Reporting::ListFilter.new(type: 'control_tag', values: [satisfies]),
```

Reply:
Do we ever provide a value for `control_tag` where this wouldn't work? Do we need to have a test demonstrating why we'd do that?

Reply:
a control tag can have a key and no value, a key and a single string value, or a key an array of string values. so the full tag filter for a control tag with a value will look like: 
```
Reporting::ListFilter.new(type: 'control_tag:scope', values: ['Apache'])
```

Reply:
and there's an example of it here: https://github.com/chef/automate/blob/master/components/compliance-service/api/tests/02_nodes_spec.rb#L1414 that was added in the pr that added the control tag filtering to the code

Reply:
That is correct, thank you Victoria.
Pete brings a good point regarding a full (key & value) control_tag filter in the suggestions tests. I added one with commit https://github.com/chef/automate/pull/1505/commits/f28bf948aa9d10eaa706cf5b7f6f955fc7a82f30

Reply:
I guess in this case I would expect all of the values to be in the values (eg, `type: 'control_tag', values: ['scope', 'Apache']`. If this is the convention the code is using this is fine then.